README.md

```
@@ -13,23 +13,36 @@
+</p>
+
+---
+
+<details>
+<summary><strong>Table of Contents</strong></summary>
+
+- [Why OpenFGA?](#why-openfga)
+- [Quickstart](#quickstart)
+- [Highlights](#highlights)
+- [Learn more](#learn-more)
+- [Installation](#installation)
```

markdownlint-cli2 (0.17.2)

Link fragments should be valid
null

(MD051, link-fragments)